mbedtls-sys/build/config.rs

```diff
@@ -31,6 +31,8 @@ pub const PREFIX: &'static str = r#"
 #ifndef MBEDTLS_CONFIG_H
 #define MBEDTLS_CONFIG_H
 
+#include <stdint.h>
```

Comment:
Is this needed because of your time_t definition? Can that be solved another way?